nodejs/gtfs-realtime.js

```diff
@@ -77,6 +86,8 @@ $root.TransitAlertExtension = (function() {
         writer.uint32(/* id 1, wireType 0 =*/8).uint64(message.createdAt);
         if (message.sourceName != null && message.hasOwnProperty("sourceName"))
             writer.uint32(/* id 2, wireType 2 =*/18).string(message.sourceName);
+        if (message.isServiceChangeAlert != null && message.hasOwnProperty("isServiceChangeAlert"))
```

Comment:
Any reason we dont do a strict comparison here? `!=` vs. `!==`

Reply:
Sorry Louis, I should have specified, the JS code in this project is automatically generated. I'll add a comment for the one actual line I changed.

I'm guessing the generated code needs to capture all nullish values (ie undefined but not 0).